pallets/xcmp-handler/src/lib.rs

```diff
@@ -403,5 +414,125 @@ pub mod pallet {
 
 			Ok(().into())
 		}
+
+		/// Get total XCMP fees for all executions of an encoded call.
```

Comment:
this seems to just be a wrapper to change the return type. Not what the comment says

Reply:
correct.  I changed it.  will remove.

Reply:
done.